internal/wasm/interpreter/interpreter.go

```diff
@@ -500,6 +500,20 @@ func (e *engine) lowerIR(ir *wazeroir.CompilationResult) (*code, error) {
 			}
 		case *wazeroir.OperationSignExtend32From8, *wazeroir.OperationSignExtend32From16, *wazeroir.OperationSignExtend64From8,
 			*wazeroir.OperationSignExtend64From16, *wazeroir.OperationSignExtend64From32:
+		case *wazeroir.OperationMemoryInit:
+			op.us = make([]uint64, 1)
+			op.us[0] = uint64(o.DataIndex)
+		case *wazeroir.OperationDataDrop:
+			op.us = make([]uint64, 1)
+			op.us[0] = uint64(o.DataIndex)
+		case *wazeroir.OperationMemoryCopy:
+		case *wazeroir.OperationMemoryFill:
+		case *wazeroir.OperationTableInit:
+			panic("TODO: table.init unimplemented")
+		case *wazeroir.OperationElemDrop:
+			panic("TODO: elem.drop unimplemented")
+		case *wazeroir.OperationTableCopy:
+			panic("TODO: table.copy unimplemented")
```

Comment:
these are the TODOs in the next PR which will complete the bulk memory proposal.